procedures/katello/branch.md.erb

```diff
@@ -1,43 +1,70 @@
-# Near-end of development phase
+# One Month Prior to Branch Date
+
+## Release Owner
 
 * [ ] Announce upcoming branching to discourse development category a month before
-* [ ] Generate a release GPG key
-* [ ] Ask plugin authors to start extracting i18n strings and pushing the changes into develop/master git branches so Transiflex can pick it up
+* [ ] Start extracting i18n strings and pushing the changes into master git branches so Transiflex can pick it up
 
-# Package build systems
+# Two Weeks Prior to Branch Date
+
+## Release Owner
 
 * [ ] Add tool_belt config for new release [tool_belt configs](https://github.com/theforeman/tool_belt/tree/master/configs/katello)
+* [ ] Ensure stable Pulp release
+  * [ ] Update the [katello-repos](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/katello/katello-repos/katello-repos.spec#L1-L2)
+  * [ ] Update [forklift](https://github.com/theforeman/forklift/blob/master/roles/katello_repositories/defaults/main.yml#L4-L5)
+
+## Release Packager
+
+* [ ] Ensure tool_belt config is merged and output from `./tools.rb koji configs/katello/<%= release %>.yaml` matches expectations
+
+# On Branch Date
+
+## Release Owner
+
+* [ ] Create KATELLO-<%= release %> branches
```

Comment:
If it's only one project then the indenting might make less sense. Anything about a hammer-cli-katello?